### ISSUE TITLE
Doesn't handle @note correctly?

### ISSUE THREAD

**Issue by virtuald** [auscompgeek/sphinxify#33]
```
  /**
   * Create a new entry for the topic, with specific type string.
   *
   * Entries act as a combination of a subscriber and a weak publisher. The
   * subscriber is active as long as the entry is not destroyed. The publisher
   * is created when the entry is first written to, and remains active until
   * either Unpublish() is called or the entry is destroyed.
   *
   * @note It is not possible to use two different data types with the same
   *     topic. Conflicts between publishers are typically resolved by the
   *     server on a first-come, first-served basis. Any published values that
   *     do not match the topic's data type are dropped (ignored), and the entry
   *     will show no new values if the data type does not match. To determine
   *     if the data type matches, use the appropriate Topic functions.
   *
   * @param typeString type string
   * @param defaultValue default value used when a default is not provided to a
   *        getter function
   * @param options publish and/or subscribe options
   * @return entry
   */
 ```

Translates to:

```
"""Create a new entry for the topic, with specific type string.

        Entries act as a combination of a subscriber and a weak publisher. The
        subscriber is active as long as the entry is not destroyed. The publisher
        is created when the entry is first written to, and remains active until
        either Unpublish() is called or the entry is destroyed.

        .. note:: I
        topic. Conflicts between publishers are typically resolved by the
        server on a first-come, first-served basis. Any published values that
        do not match the topic's data type are dropped (ignored), and the entry
        will show no new values if the data type does not match. To determine
        if the data type matches, use the appropriate Topic functions.

        :param typeString:   type string
        :param defaultValue: default value used when a default is not provided to a
                             getter function
        :param options:      publish and/or subscribe options

        :returns: entry
        """
        ```

**Comment by virtuald** [comment]
Still doesn't work, here's a simpler version:

```
/**
 * NetworkTables struct-encoded value entry.
 *
 * @note Unlike NetworkTableEntry, the entry goes away when this is destroyed.
 */
```

```
        """NetworkTables struct-encoded value entry.

        .. note:: U
        """
```

**Comment by auscompgeek** [comment]
Fixed in 0.11.
